bin/crucible

```diff
@@ -138,7 +135,7 @@ if [ "$CRUCIBLE_USE_LOGGER" == "1" ]; then
 fi
 
 # invoke the main command handler
-${CRUCIBLE_HOME}/bin/_main "${LOGGER_SESSION_ID}" "${LOG_DB}" "$@"
+${CRUCIBLE_HOME}/bin/_main "${LOG_DB}" "$@"
```

Comment:
why? session_id is not needed ?

Reply:
It is now being generated and exported in `bin/base` which is sourced by almost all our scripts, so passing it via the CLI just seemed redundant.